Gemfile

```diff
@@ -64,6 +64,7 @@ gem 'lograge', '~> 0.11.2'
 gem 'okcomputer'
 gem 'pg'
 gem 'propshaft'
+gem 'pry-rails'
```

Comment:
This project is not actively maintained. It doesn't have tests for Rails 7 and it has no CI builds. See https://github.com/pry/pry-rails#readme  Are we sure this is a good dependency to add?

Reply:
Well, that's not good.  I definitely ran into some issues with irb when doing debugging work on the rails console for #2593 that worked fine with pry, but perhaps team discussion is warranted (or at minimum putting in the development group here).